fenix/app/src/main/res/values/strings.xml

```diff
@@ -204,6 +204,8 @@
     <string name="resync_button_content_description">Resync</string>
     <!-- Browser menu button that opens the find in page menu -->
     <string name="browser_menu_find_in_page">Find in page</string>
+    <!-- Browser menu button that opens translations flow. -->
```

Comment:
nit: Maybe just a little more detail since it is a new feature? Something like: 
```suggestion
    <!-- Browser menu button that opens the translations dialog, which has options to translate the current browser page. -->
```

Reply:
Done